CHANGELOG.md

```diff
@@ -6,6 +6,59 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release.
 
+## [v2.1.0-rc1][v2.1.0-rc1]
```

Comment:
Optional changelog word-smithing can go in here as well already.

Reply:
Done.